pkgs/development/compilers/llvm/common/llvm/default.nix

```diff
@@ -399,6 +402,8 @@ stdenv.mkDerivation (rec {
         nativeInstallFlags
       ])
     )
+  ] ++ optionals enableDebuginfod [
+    "-DLLVM_ENABLE_CURL=ON"
```

Comment:
Should use one of the CMake option functions in lib

Reply:
I modeled this according to how it's done in the rest of the file. Should I just break the convention?

Reply:
My 2¢: I think it's better how roblabla originally wrote it. For one thing, you get back exactly the original derivation by passing an override of `enableDebuginfod = false;`, whereas if you pass `LLVM_ENABLE_CURL=OFF`, it's an unnecessary rebuild. Not to mention it's consistent with the code as currently written, and there is no indirection involved (so no other non-local code to read if you need to be certain about what is happening).